packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3492,6 +3506,36 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.members.admitMembers(memberIds, locusUrls);
   }
 
+  /**
+   * Manages be right back status updates for the current participant.
+   *
+   * @param {boolean} enabled - Indicates whether the use is brb or not.
+   * @returns {Promise<void>} - A promise that resolves when the request is complete.
+   * @throws {Error} - Throws an error if the request fails.
+   */
+  public beRightBack(enabled: boolean) {
+    return this.meetingRequest
+      .sendBrb({
+        enabled,
+        locusUrl: this.locusUrl,
+        deviceUrl: this.deviceUrl,
+        selfId: this.selfId,
+      })
+      .then((payload) => {
+        if (this.isMultistream && this.mediaProperties.webrtcMediaConnection) {
```

Comment:
just to double check: what about non-multistream cases? do we support this feature for 1-1 calls (that's the only type of calls webex web app currently has that are still transcoded). Do we need to signal anything to Edonus via the SDP or some other way? I guess not, but I just want to be sure, maybe it's worth asking the Edonus team about this.

Reply:
Created jira for that: [SPARK-578667](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-578667)